web/public/img/bar_zone1.svg

```diff
@@ -0,0 +1,22 @@
+<svg width="255px" height="266px" viewBox="0 0 255 266" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Please optimize the SVG using https://jakearchibald.github.io/svgomg/. Make sure you group the things that belong together to make animations and adding click events easier.